packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -505,8 +505,8 @@ export const defaultError = errorMap[``]
 
 export interface IErrorMapEntry {
   text: (context) => string
-  level: Level
-  type?: Type
+  level: keyof typeof Level
```

Comment:
It would be good to add a comment explaining why this is needed, as it looks odd.